langchain4j-core/src/main/java/dev/langchain4j/model/output/TokenUsage.java

```diff
@@ -12,6 +12,8 @@ public class TokenUsage {
     private final Integer inputTokenCount;
     private final Integer outputTokenCount;
     private final Integer totalTokenCount;
+    private final Integer cacheCreationInputTokens;
```

Comment:
This is an Anthropic-specific information, what do you think about creating `AnthropicTokenUsage` that extends `TokenUsage` and add those fields there?

Reply:
Created AnthropicTokenUsage